docs/tutorials.state_exclusion.rst

```diff
@@ -206,6 +207,58 @@ The SDP not only gives us the optimal value, which is $0$ in this case, indicati
 antidistinguishable, but we also get a set of optimal measurement operators. These should look familiar to the
 measurements we explicitly constructed earlier.
 
+
+Antidistinguishability and :math:`(n-1)`-incoherence
+----------------------------------------------------
+
+Antidistinguishability of a set of pure states is equivalent to a certain notion from the theory of quantum resources
+referred to as :math:`k`-incoherence :cite:`Johnston_2022_Absolutely`:
+
+**Definition**: Let :math:`k` be a positive integer. Then :math:`X \in \text{Pos}(\mathbb{C} ^n)` is called
```

Comment:
You should define what $n$ is here. Is it the number of states or the dimension?

Reply:
I'll add that it's a positive integer, but I think it's implicit based on how it is used in the definition that it corresponds to the dimension of the space.